presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -921,6 +926,12 @@ public void testBinaryFunctions()
         assertQuery("SELECT from_ieee754_64(to_ieee754_64(-3.14158999999999988261834005243E0))");
         assertQuery("SELECT from_ieee754_64(to_ieee754_64(totalprice)) FROM orders");
         assertQuery("SELECT from_ieee754_64(to_ieee754_64(acctbal)) FROM customer");
+
+        //secure_random
+        assertQuerySucceeds("SELECT secure_random(1, 10)");
```

Comment:
Please make new test methods.

Reply:
Thank you. Will create separated test method for each function.